modules/Bio/EnsEMBL/Analysis/Hive/Config/Projection_subpipeline_conf.pm

```diff
@@ -0,0 +1,518 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::Analysis::Hive::Config::Projection_subpipeline_conf;
+
+use strict;
+use warnings;
+use File::Spec::Functions;
+
+use Bio::EnsEMBL::ApiVersion qw/software_version/;
+use Bio::EnsEMBL::Analysis::Tools::Utilities qw(get_analysis_settings);
+use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
+use base ('Bio::EnsEMBL::Analysis::Hive::Config::HiveBaseConfig_conf');
+
+sub default_options {
+  my ($self) = @_;
+  return {
+    # inherit other stuff from the base class
+    %{ $self->SUPER::default_options() },
+
+######################################################
+#
+# Variable settings- You change these!!!
+#
+######################################################
+########################
+# Misc setup info
+########################
+    'dbowner'                   => '' || $ENV{EHIVE_USER} || $ENV{USER},
+    'pipeline_name'             => '' || $self->o('production_name').'_'.$self->o('ensembl_release'),
+    'production_name'           => '', # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'release_number'            => '' || $self->o('ensembl_release'),
+    'user_r'                    => '', # read only db user
+    'user'                      => '', # write db user
+    'password'                  => '', # password for write db user
+    'pipe_db_server'            => '', # host for pipe db
+    'dna_db_server'             => '', # host for dna db
+    'databases_server'          => '', # host for general output dbs
+    'pipe_db_port'              => '', # port for pipeline host
+    'dna_db_port'               => '', # port for dna db host
+    'databases_port'            => '', # port for general output db host
+    'uniprot_set'               => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'output_path'               => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'use_genome_flatfile'       => '1',# This will read sequence where possible from a dumped flatfile instead of the core db
+    'skip_projection'           => '0', # Will skip projection process if 1
+
+########################
+# Pipe and ref db info
+########################
+
+    'projection_source_db_name'         => '', # This is generally a pre-existing db, like the current human/mouse core for example
+    'projection_source_db_server'       => 'mysql-ens-mirror-1',
+    'projection_source_db_port'         => '4240',
+    'projection_source_production_name' => '',
+
+    'compara_db_name'   => 'leanne_ensembl_compara_95',
+    'compara_db_server' => 'mysql-ens-genebuild-prod-5',
+    'compara_db_port'   => 4531,
+
```

Comment:
This can be removed or used to add selected_projection_db_(server|port) for the selected_projection_db below

Reply:
Removed